pkg/storage/factory_test.go

```diff
@@ -50,23 +54,65 @@ func TestFactoryStop(t *testing.T) {
 	store.Stop()
 }
 
+func TestCassandraInMultipleSchemas(t *testing.T) {
```

Comment:
I'm curious why Cassandra is being tested specifically here?

Reply:
not sure, I didn't touch this
must have moved it when adding new tests